test/source.js

```diff
@@ -29,10 +30,27 @@ function testAllSources(validate) {
       t.end();
     });
     tape(`${source} filename field must have a matching file in the data directory`, (t) => {
-      const data = Hjson.parse(fs.readFileSync(source, 'utf8'));
       const sourceFilename = data.filename;
       t.ok(fs.existsSync(`./data/${sourceFilename}`));
       t.end();
     });
+    if (data.conform.type === 'geojson') {
+      tape(`${source} data must be valid and simple`, (t) => {
+        const geojson = fs.readFileSync(`./data/${data.filename}`, 'utf8');
+        validateGeometry(geojson, t);
+      });
+    }
   });
 }
+
+function validateGeometry(geojson, t) {
+  const reader = new jsts.io.GeoJSONReader();
+  const fc = reader.read(geojson);
+  t.ok(fc.features.every(feat => {
```

Comment:
minor simplification here and the same below.
```suggestion
  t.ok(fc.features.every(feat => feat.geometry.isSimple()
```